agents/template/src/main/scala/org/broadinstitute/transporter/kafka/TransferStreamBuilder.scala

```diff
@@ -18,9 +18,9 @@ import org.broadinstitute.transporter.transfer.{
   TransferRunner
 }
 
-class TransferStreamBuilder[In: Decoder, Progress: Encoder: Decoder, Out: Encoder](
+class TransferStreamBuilder[I: Decoder, P: Encoder: Decoder, O: Encoder](
```

Comment:
Just curious... why I, P and O instead of In, Progress and Out?

Reply:
We were having some issues with shadowing type names, this helped alleviate that issue 🤷‍♂

Reply:
This is also more idiomatic Scala (sadly)